tests/unit/CMakeLists.txt

```diff
@@ -1052,10 +1052,17 @@ add_executable(ot-test-timer
 add_executable(ot-test-toolchain
     test_toolchain.cpp test_toolchain_c.c
 )
+
 target_include_directories(ot-test-toolchain
     PRIVATE
         ${COMMON_INCLUDES}
 )
+
+target_link_libraries(ot-test-toolchain
```

Comment:
Is this a related change?

Reply:
Yes. Now that we have `topology.cpp` file (added in this PR) we need to include the `COMMON_LIBS` when building all unit tests. We do this for other tests but I guess wanted to keep `ot-test-toolchain` simple.